packages/@tinacms/cli/src/buildTina/index.ts

```diff
@@ -0,0 +1,208 @@
+  next: () => void,
+  opts: BuildSetupOptions
+) => {
+  const rootPath = ctx.rootPath as string
+  const { bridge, database, store } = await buildSetup({
+    ...opts,
+    rootPath,
+    useMemoryStore: false,
+  })
+  // attach to context
+  ctx.bridge = bridge
+  ctx.database = database
+  ctx.store = store
+
+  next()
+}
+
+const buildSetup = async ({
+  isomorphicGitBridge,
+  experimentalData,
+  rootPath,
+  useMemoryStore,
+}: BuildSetupOptions & { rootPath: string; useMemoryStore: boolean }) => {
+  const fsBridge = new FilesystemBridge(rootPath)
+  const isomorphicOptions =
+    isomorphicGitBridge && (await makeIsomorphicOptions(fsBridge))
+
+  /**
+   * To work with Github directly, replace the Bridge and Store
+   * and ensure you've provided your access token.
+   * NOTE: when talking the the tinacms repo, you must
+   * give your personal access token access to the TinaCMS org
+   */
+  // const ghConfig = {
+  //   rootPath: 'examples/tina-cloud-starter',
+  //   accessToken: '<my-token>',
+  //   owner: 'tinacms',
+  //   repo: 'tinacms',
+  //   ref: 'add-data-store',
+  // }
+  // const bridge = new GithubBridge(ghConfig)
+  // const store = new GithubStore(ghConfig)
+
+  const bridge = isomorphicGitBridge
+    ? new IsomorphicBridge(rootPath, isomorphicOptions)
+    : fsBridge
+
+  // const store = experimentalData
+  //   ? new LevelStore(rootPath, useMemoryStore)
+  //   : new FilesystemStore({ rootPath })
+
+  const store = new LevelStore(rootPath, useMemoryStore)
+
+  const database = await createDatabase({ store, bridge })
+
+  return { database, bridge, store }
+}
+
+export const buildCmdBuild = async (
+  ctx: any,
+  next: () => void,
+  options: Omit<
+    BuildOptions & BuildSetupOptions,
+    'bridge' | 'database' | 'store'
+  >
+) => {
+  const bridge: Bridge = ctx.bridge
+  const database: Database = ctx.database
+  const store: Store = ctx.store
+  // always skip indexing in the "build" command
+  await build({
+    ...options,
+    bridge,
+    database,
+    store,
+    ctx: ctx,
+    skipIndex: true,
+  })
+  next()
+}
+
+export const build = async ({
+  noWatch,
+  ctx,
+  bridge,
+  database,
+  store,
+  beforeBuild,
+  afterBuild,
+  dev,
+  isomorphicGitBridge,
+  local,
+  verbose,
+  noSDK,
+  skipIndex,
+}: BuildOptions) => {
+  const rootPath = ctx.rootPath as string
+  if (!rootPath) {
+    throw new Error('Root path has not been attached')
+  }
+  // Clear the cache of the DB passed to the GQL server
+  database.clearCache()
+
+  if (beforeBuild) {
+    await beforeBuild()
+  }
+
+  try {
+    if (!process.env.CI && !noWatch) {
+      await store.close()
+      await resetGeneratedFolder({
+        tinaGeneratedPath: path.join(rootPath, '.tina', '__generated__'),
+      })
+      await store.open()
+    }
+    const cliFlags = []
+    // always enable experimentalData and isomorphicGitBridge on the  backend
+    cliFlags.push('experimentalData')
+    cliFlags.push('isomorphicGitBridge')
```

Comment:
the isomorphicGitBridge BuildOption is not used - is that intentional? Should it be removed from BuildOptions?

also why are the flags being populated regardless of the actual cli args? There is already a PR in tina cloud to set these based on the version of @tinacms/graphql

Reply:
We thought we could do these for the beta release so that folks could try it out with `yarn add @tinacms/cli@beta`.

We can remove it after thought.

Reply:
right, I forgot about the beta plan 👍🏻